channeld/channeld.c

```diff
@@ -2849,6 +2884,9 @@ static void init_channel(struct peer *peer)
 	if (funding_signed)
 		sync_crypto_write(&peer->cs, PEER_FD, take(funding_signed));
 
+	/* from now we need keep watch over WIRE_CHANNEL_FUNDING_DEPTH */
+	peer->depth_togo = minimum_depth;
```

Comment:
Maybe this line should be moved to before `if (reconnected)` line 2880?
because `peer_reconnect` calls `billboard_update` indirectly, maybe it is safer to have `depth_togo` initialized?

Reply:
Yes!
`peer_reconnect` calls `peer_bilboard`(it has some difference with `bilboard_update`), but it's better to initial `depth_togo` ASAP!